GunplaOnlineShop/GunplaOnlineShop/Views/Account/Register.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model GunplaOnlineShop.ViewModels.RegisterViewModel
+@{
+    ViewBag.title = "Register";
+}
+<link href="~/css/signin-form.css" rel="stylesheet" />
+<div class="signup-form">
+    <form method="post" class="form-horizontal">
+        <div class="col-xs-8 col-xs-offset-4">
+            <h2>Sign Up</h2>
+        </div>
+        <div asp-validation-summary="All" class="text-danger"></div>
+        <div class="form-group">
+            <label asp-for="Email" class="control-label col-xs-4">Email Address</label>
```

Comment:
No need to type `Email Address` here
Use the `[Display(Name = "Email Address")]` data annotation attribute to decorate the property combine with `asp-for`, it will auto generate the html element with corresponding attributes